src/figuresClasses.ts

```diff
@@ -1,19 +1,88 @@
+type Color = 'red' | 'green' | 'blue';
+type Shape = 'triangle' | 'circle' | 'rectangle';
+
 export interface Figure {
+  color: Color,
+  shape: Shape,
 
+  getArea(): number,
 }
 
-export class Triangle {
-
+function isCompare(a: number, b: number, c: number): boolean {
+  return a + b <= c || b + c <= a || c + a <= b;
 }
 
-export class Circle {
+export class Triangle implements Figure {
+  shape = 'triangle' as Shape;
+
+  color: Color;
+
+  constructor(
+    color: Color,
+    private a: number,
+    private b: number,
+    private c: number,
+  ) {
+    this.color = color;
 
+    const isBigger = isCompare(this.a, this.b, this.c);
+
+    if (this.a <= 0 || this.b <= 0 || this.c <= 0 || isBigger) {
+      throw new Error('Error -:)');
```

Comment:
<img width="1043" alt="image" src="https://github.com/mate-academy/ts_figures_classes/assets/86789161/81ba5142-ca3b-41ed-88f6-7da327f7c1bf">

The same comment to all other error messages